shared/api/network.go

```diff
@@ -58,3 +58,23 @@ type NetworkLease struct {
 	Address  string `json:"address" yaml:"address"`
 	Type     string `json:"type" yaml:"type"`
 }
+
+// NetworkStateStats represents statistics about an interface
+//
+// API extension: network_state
+type NetworkStateStats struct {
```

Comment:
This should be renamed to NetworkStateCounters for consistency with containerStateNetworkCounters and its fields should also be renamed to match the container equivalent of this:

```golang
type ContainerStateNetworkCounters struct {
    BytesReceived   int64 `json:"bytes_received" yaml:"bytes_received"`
    BytesSent       int64 `json:"bytes_sent" yaml:"bytes_sent"`
    PacketsReceived int64 `json:"packets_received" yaml:"packets_received"`
    PacketsSent     int64 `json:"packets_sent" yaml:"packets_sent"`
}
```